src/openrct2-ui/windows/RideConstruction.cpp

```diff
@@ -2928,7 +2928,12 @@ static Widget _rideConstructionWidgets[] = {
         {
             if (im.IsModifierKeyPressed(ModifierKey::ctrl))
             {
-                auto info = GetMapCoordinatesFromPos(screenCoords, 0xFCCA);
+                auto interactionFlags = EnumsToFlags(
```

Comment:
Is this correct i count 7 enums and 0xFCCA is 10 set bits.

Reply:
Yes, as there are only 13 interaction types:
```cpp
enum class ViewportInteractionItem : uint8_t
{
    None,
    Terrain,
    Entity,
    Ride,
    Water,
    Scenery,
    Footpath,
    PathAddition,
    ParkEntrance,
    Wall,
    LargeScenery,
    Label,
    Banner
};
```

With 0xFCCA == 0b1111110011001010, that excludes None, Entity, Water, Scenery, ParkEntrance, Wall. The three upper bits don't actually represent an interaction type.